doc/tutorial/staramr-tutorial.ipynb

```diff
@@ -264,6 +307,26 @@
     "cut -f 1,3 out/summary.tsv | column -s$'\\t' -t"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Isolate ID       Plasmid Genes\n",
+      "GCF_001478105.1  IncI1, IncX1\n",
+      "GCF_001931595.1  None\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Show only Plasmid Genes\n",
```

Comment:
In the description right below this section could you add a sentence about how this shows the plasmid genes, similar to the description for the AMR genes?

>This file summary.tsv contains a summary of all the results in a single table, one genome per line. According to these results, the genomes GCF_001478105.1 and GCF_001931595.1 contain the listed AMR genes under Genotype and are resistant to the listed drugs under Predicted Phenotype.

Reply:
Solved in commit 2202e45